src/victory-util/label-helpers.js

```diff
@@ -1,162 +1,160 @@
-import Helpers from "./helpers";
+import { evaluateProp, scalePoint, degreesToRadians, radiansToDegrees, getPoint } from "./helpers";
```

Comment:
I think @boygirl will have to give the final word on this, but I think the convention is to split these kinds of import statements onto multilpe lines, i.e.

```javascript
import {
  evaluateProp,
  scalePoint,
  degreesToRadians,
  radiansToDegrees,
  getPoint
} from "./helpers";